src/sparql/ordo-replace-annotation-based-mappings.ru.jinja2

```diff
@@ -0,0 +1,28 @@
+prefix skos: <http://www.w3.org/2004/02/skos/core#>
+prefix ECO: <http://purl.obolibrary.org/obo/ECO_>
+prefix owl: <http://www.w3.org/2002/07/owl#>
+prefix oboInOwl: <http://www.geneontology.org/formats/oboInOwl#>
+prefix sssom: <https://mapping-commons.github.io/sssom/purl/>
```

Comment:
I have a problem related to the `sssom` prefix.

In the output `.owl` file generated, the prefix is showing at the top as this:
`     xmlns:purl="https://mapping-commons.github.io/sssom/purl/"`
I was hoping it would output this:
`     xmlns:sssom="https://mapping-commons.github.io/sssom/purl/"`

Here's what some references look like in the output `.owl` file (I copy pasted from separate places in the file):
```owl
<!-- https://mapping-commons.github.io/sssom/purl/notExactMatch -->

<owl:AnnotationProperty rdf:about="https://mapping-commons.github.io/sssom/purl/notExactMatch"/>

<purl:notExactMatch>OMIM:602782</purl:notExactMatch>
```

## Quesitons
1. Does this merit making a GitHub issue to remember to fix this?
2. How do I/we fix this?
a. Does this happen before `mondo-ingest`? (I'm guessing not)
b. If not (a), then where should this be done? During the makefile recipe which runs a bunch of SPARQL updates? Or somewhere else?

Reply:
Both are wrong! See https://github.com/mapping-commons/sssom/blob/master/src/linkml/sssom.yaml#L8

Reply:
excellent